virt/storage/virt-automatic-bootsource-updates.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+[id="virt-automatic-bootsource-updates"]
+= Managing automatic boot source updates
+include::_attributes/common-attributes.adoc[]
+:context: virt-automatic-bootsource-updates
+
+toc::[]
+
+You can manage automatic updates for the following boot sources:
+
+* xref:../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[All Red Hat boot sources].
+* xref:../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-custom-boot-source-updates_virt-automatic-bootsource-updates[All custom boot sources].
+* xref:../../virt/storage/virt-automatic-bootsource-updates.adoc#virt-disable-auto-updates-single-boot-source_virt-automatic-bootsource-updates[Individual Red Hat or custom boot sources].
+
+Boot sources can make virtual machine (VM) creation more accessible and efficient for users. If automatic boot source updates are enabled, the Containerized Data Importer (CDI) imports, polls, and updates the images so that they are ready to be cloned for new VMs. By default, CDI automatically updates Red Hat boot sources.
```

Comment:
```suggestion
Boot sources can make virtual machine (VM) creation more accessible and efficient for users. If automatic boot source updates are enabled, the Containerized Data Importer (CDI) imports, polls, and updates the images so that they are ready to be cloned for new VMs. By default, the CDI automatically updates Red Hat boot sources.
```